Dockerfile.template

```diff
@@ -0,0 +1,27 @@
+FROM balenalib/%%BALENA_MACHINE_NAME%%-node AS npm-install
+
+RUN apt-get update && \
+    apt-get install -y git python build-essential && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /tmp/node
+
+COPY package.json .
+
+RUN npm install
+
+FROM balenalib/%%BALENA_MACHINE_NAME%%-node
+
+ENV UDEV=1
+
+WORKDIR /usr/app
+
+COPY --from=npm-install /tmp/node ./
+
+COPY tsconfig.json .
+COPY typings typings
+COPY lib lib
```

Comment:
Is it not worth building the code in the stage above, and just bringing node_modules and the build directory down into this stage?